R/brreg-utils.R

```diff
@@ -176,3 +172,22 @@ mod11 <- function(x, n = 8, weights = c(3,2,7,6,5,4,3,2)) {
   k <- sum(weights * x[1:n]) %% 11
   if (k == 0) k == x[n+1] else (11 - k) == x[n+1]
 }
+
+#' paste_brreg_address
+#' @noRd
+paste_brreg_address <- function(x) {
+  address <- unlist(x$adresse)
+  n <- length(address)
+  if (!'postnummer' %in% names(x)) x$postnummer <- ''
+  if (!'poststed' %in% names(x)) x$poststed <- ''
+  if (is.null(address)) {
+    out <- sprintf('%s %s', x$postnummer, x$poststed)
+  } else if (n == 1) {
+    out <- sprintf('%s, %s %s', address, x$postnummer, x$poststed)
+  } else {
+    address <- paste(address, collapse = ', ')
+    out <- sprintf('%s, %s %s', address, x$postnummer, x$poststed)
+  }
+  out <- gsub('  +', ' ', out)
```

Comment:
Are we always looking for multiple whitespace (`\u0020`), or should we catch _all_ whitespace with `gsub('\\s+', ' ', out)`?

Reply:
Yep. I think we are looking for mulitiple whitespace (singles should be kept). But using unicode is definitely better.

Reply:
Resolved with 0b993fd